### PR TITLE
adding recipes views and test

### PR DESCRIPTION
**Pull request by isabelrios**

### CODE REVIEW

**Comment by isabelrios** [comment]
@JohanLorenzo please r? 
Now we will have one more test and more views helpful to add more tests in the future

File: tests/common/new_recipe_test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  afterEach(() => {
+    return app.cleanUp();
+  });
+
+  after(() => {
+    return app.stop();
+  });
+
+  it('should be able to create a new recipe', () => {
```
Comment (JohanLorenzo) [inline tests/common/new_recipe_test.js]: Nit: Rename this test as we don't actually create the recipe

File: tests/common/new_recipe_test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    return servicesGeneralView.goToRecipesView()
+     .then(recipesView => {
+       return recipesView.tapCreateNewRecipe(); });
```
Comment (JohanLorenzo) [inline tests/common/new_recipe_test.js]: Nit: `recipesView.goToNewRecipe()`. `tap` is a detail of how a user fulfill his intention

File: tests/lib/views/new_recipe/view.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+var View = require('../view');
+var NewRecipeAccessors = require('./accessors');
+
+
+function NewRecipeView() {
+  [].push.call(arguments, NewRecipeAccessors);
+  View.apply(this, arguments);
+
+  this.accessors.newRecipePage; // Wait until it appears
```
Comment (JohanLorenzo) [inline tests/lib/views/new_recipe/view.js]: This line should fail, at the moment `newRecipePage` is not defined.

File: tests/lib/views/new_recipe/accessors.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+var Accessors = require('../accessors');
+
+
+function NewRecipeAccessors() {
+  Accessors.apply(this, arguments);
+}
+
+NewRecipeAccessors.prototype = Object.assign({
+  //to add functions here
```
Comment (JohanLorenzo) [inline tests/lib/views/new_recipe/accessors.js]: I'd wait on a locator here, so make sure that this page is actually loaded.

**Comment by JohanLorenzo** [comment]
Looks good to me, modulo [this change](https://github.com/fxbox/app/pull/102/files#r60070409) and a few nits. r+

**Pull request merged**